config/_default/menus/menus.en.toml

```diff
@@ -56,18 +56,18 @@
     weight = 400
 [[footer]]
     parent = "column1"
-    name = "Documentation"
-    url = "/docs/"
+    name = "Download"
+    url = "/docs/install/"
```

Comment:
/download.  This should be the URL when the user presses the "Download" button on the Header Nav Bar.